src/components/NewCommentForm/NewCommentForm.jsx

```diff
@@ -1,39 +1,84 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import './NewCommentForm.scss';
 
-export const NewCommentForm = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+export const NewCommentForm = ({ postId, createComment }) => {
+  const [nameOfAuthor, setName] = useState('');
+  const [emailOfAuthor, setEmail] = useState('');
+  const [bodyOfComment, setBodyOfComment] = useState('');
+
+  const handleChangeName = ({ target }) => {
+    const { value } = target;
+
+    setName(value);
+  };
+
+  const handleChangeEmail = ({ target }) => {
+    const { value } = target;
+
+    setEmail(value);
+  };
+
+  const handleChangeBody = ({ target }) => {
+    const { value } = target;
+
+    setBodyOfComment(value);
+  };
+
+  return (
+    <form
+      className="NewCommentForm"
+      onSubmit={(e) => {
+        e.preventDefault();
+        createComment(nameOfAuthor, emailOfAuthor, bodyOfComment, postId);
+        setName('');
+        setEmail('');
+        setBodyOfComment('');
+      }}
     >
-      Add a comment
-    </button>
-  </form>
-);
+      <div className="form-field">
+        <input
+          value={nameOfAuthor}
+          onChange={e => handleChangeName(e)}
+          type="text"
+          name="name"
+          placeholder="Your name"
+          className="NewCommentForm__input"
+        />
+      </div>
+
+      <div className="form-field">
+        <input
+          value={emailOfAuthor}
+          onChange={e => handleChangeEmail(e)}
```

Comment:
do the same for other similar functions

```suggestion
          onChange={e => setEmail(e.target.value)}
```